ark/phenotyping/som_utils.py

```diff
@@ -619,10 +678,8 @@ def cluster_pixels(fovs, base_dir, pre_dir='pixel_mat_preprocessed',
     misc_utils.verify_in_list(provided_fovs=fovs,
                               subsetted_fovs=io_utils.remove_file_extensions(files))
 
-    # precompute row sums for each fov (more efficient in Python than R)
     print("Normalizing row sums and removing rows that sum to 0")
```

Comment:
This print statement seems out of place to me.

Reply:
The idea of the print statements is to provide the user with progress updates throughout. If this step takes a long time, I think it makes sense to leave it. If it's pretty quick, it can be removed.

Reply:
Right, I just don't think the normalizing row sums and removing rows that sum to 0 is actually happening here. I think maybe some refactoring happened and Alex forgot to remove this print statement?

Reply:
@cliu72 good point, that statement was there because we used to do separate row sum normalizations for the subsetted and preprocessed data. Now, we handle it all in `create_pixel_matrix`.